src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -196,10 +251,9 @@ struct TimeDependentMapOptions {
    * - Shape: `Shape` (with a size function of time)
    * - Translation: `Translation`
    */
-  void build_maps(
-      const std::array<double, 3>& center, double inner_radius,
-      double outer_radius,
-      std::pair<double, double> translation_transition_radii);
+  void build_maps(const std::array<double, 3>& center, double inner_radius,
+                  double outer_radius,
+                  std::pair<double, double> translation_transition_radii);
```

Comment:
You probably only need one `transition_radius` since the outer boundary radius is already specified. Also this transition radius also applies for expansion, so maybe remove the `translation_` part.

Reply:
Not sure how to remove this. The outer radius variable there is for the shape map. It seems to mean the outer radius of the first shell. Still need the outer radius of the whole sphere domain.

Reply:
This notation seems fine. Implemented